src/styles/login.scss

```diff
@@ -0,0 +1,154 @@
+.login {
+  display: flex;
+  flex-direction: column;
+  background-color: #ffffff;
+}
+h3 {
+  color: black;
+}
+.login__nav a {
+  font-size: 25px;
+}
+.login__intro{
+  margin: 10px 0;
+}
+.login__intro p {
+  color: #999595;
+}
+.login__input__field {
+  margin: 20px 0;
+}
+.login__input__field label {
+  color: #464444;
+}
+.login__input__field input {
+  height: 35px;
+  background: rgba(196, 196, 196, 0.21);
+  border: unset;
+  border-radius: unset;
+}
+.login__input__field input:focus {
+  background: rgba(196, 196, 196, 0.21);
+  outline: none;
+}
+.login__btn--large {
+  background: #66008C;
+  color: #ffffff;
+  height: 45px;
+  box-shadow: 2px 2px 4px 0px rgba(0,0,0,0.3);
+  border: 1px solid #66008c;
+  margin-top: 20px;
+  border-radius: unset;
+}
+.login__socials__intro {
+  margin: 20px 0 0;
+  position: relative;
+  text-align: center;
+}
+.login__socials__intro hr {
+  width: 100%;
+  top: 0;
+}
+.login__socials__intro p {
+  position: absolute;
+  top: 4px;
+  width: 50%;
+  left: 25%;
+  background: #ffffff;
+  right: 25%;
+}
+.login__forgot-password {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+.login__forgot-password a {
+  font-size: 12px;
+  color: #999595;
+}
```

Comment:
@thislekan why are these lines of code commented out?

Reply:
@Andraquin I think you are pointing to the wrong lines of code. For the commented out code, it is for the social login functionality. They are commented out because they could be useful for Harry. If he wants to use it, fine. If not, he can decide to delete.